File: Backend/src/main/java/com/casaAgenda/dto/booking/BookingResponseDto.java
package com.casaAgenda.dto.booking;

import com.casaAgenda.enums.Status;

import java.time.LocalDate;

public record BookingResponseDto(Long id,
                                 Long apartmentId,
                                 LocalDate checkIn,
                                 LocalDate checkOut,
                                 String guestName,
                                 Integer people,
                                 Double deposit,
                                 Double remaining,
                                 Double total,
                                 Status status,
                                 String notes,
                                 Double priceNight,
                                 Double pricePerPerson
                                 ) {
}

File: Frontend/CasaAgenda/src/features/bookings/dtos/bookingResponseDto.ts
import { Status } from '../enum/status';

export interface BookingResponseDto {
  id: number;
  apartmentId: number;
  checkIn: string;
  checkOut: string;
  guestName: string;
  people: number;
  deposit: number;
  remaining: number;
  total: number;
  status: Status;
  notes: string;
  priceNight: number;
  pricePerPerson: number;
}
